src/ui/React/LogBoxManager.tsx

```diff
@@ -249,7 +249,12 @@ function LogWindow(props: IProps): React.ReactElement {
 
   function title(full = false): string {
     const maxLength = 30;
-    const t = `${script.filename} ${script.args.map((x: ScriptArg): string => `${x}`).join(" ")}`;
+    const server = GetServer(script.server);
+    let hostname = "";
+    if (server !== null) {
+      hostname = `${server.hostname}: `;
+    }
+    const t = `${hostname}${script.filename} ${script.args.map((x: ScriptArg): string => `${x}`).join(" ")}`;
```

Comment:
```suggestion
    const t = `${script.server}: ${script.filename} ${script.args.join(" ")}`;
```
It's pre-existing code, but that map function to turn arguments into strings is also not needed - [joining them](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join#description) already does that.